.azure-pipelines/win32-conda-CI.yml

```diff
@@ -46,13 +46,20 @@ jobs:
       echo Test numpy installation... && python -c "import numpy"
       pip install %COREML_PATH% %ONNX_PATH%
       pip install -r requirements-dev.txt
+      pip install onnxruntime==0.1.4
       echo Test onnxruntime installation... && python -c "import onnxruntime"
```

Comment:
I didn't find an onnxruntime install in the Win32 CI build, although it checks for it in line 50, so I added it above. Is this assumption correct?